QUARKUS-1397.md

```diff
@@ -0,0 +1,31 @@
+# Quarkus-1397 - Dev preview for Java 17 in Native mode
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1397
+
+This test plan cover JDK17 Mandrel dev preview Quarkus integration.
+
+## Related test plans
+
+ - [QUARKUS-728](QUARKUS-728)
+
+## Scope of the testing
+
+Test development will focus on
+- Current native scenarios must work with Java 17
+
+### Impact on testsuites and testing automation:
+
+- Native daily GitHub builds will cover Mandrel community JDK17 container image `ubi-quarkus-mandrel:21.3-java17`
+- Mandrel image based on Java 17 (`quarkus/mandrel-21-jdk17-rhel8`) is in a technical preview stage and is going to be covered by extended platform testing as a subset of testing coverage.
+- Native JDK17 will be covered by the following Jenkins jobs:
+  - rhbq-2.7-extended-platform-testing-trigger
+     - rhbq-2.7-baremetal-ts-native
+     - rhbq-2.7-rhel8-openshift-ts-jvm-ocp-native
```

Comment:
I don't think we need this, Mandrel based on Java 17 would be handled by baremetal job.

We just need make sure to do some checks with the Java 17 based image and OpenShift as part of the initial experiments. This should be covered in "Getting familiar with the feature"

Reply:
Do you mean that we could remove this paragraph? 
```
- Native JDK17 will be covered by the following Jenkins jobs:
  - rhbq-2.7-extended-platform-testing-trigger
     - rhbq-2.7-baremetal-ts-native
     - rhbq-2.7-rhel8-openshift-ts-jvm-ocp-native
```

And just mention in `Getting familiar with the feature` that we are going to do some "experiment" though this jobs?